src/systems/breadcrumbs/Breadcrumbs.cc

```diff
@@ -385,6 +400,30 @@ void Breadcrumbs::OnDeploy(const msgs::Empty &)
   IGN_PROFILE("Breadcrumbs::PreUpdate");
   {
     std::lock_guard<std::mutex> lock(this->pendingCmdsMutex);
+
+    // Check topic statistics for dropped messages
+    if (this->topicStatistics)
```

Comment:
Can this be done after the mutex is released?

Reply:
Moved in 8f861132968765420150dfa6d7e165c7aa4b7dc4